regolith/helpers/l_progressreporthelper.py

```diff
@@ -0,0 +1,191 @@
+    btype = HELPER_TARGET
+    needed_dbs = [f'{TARGET_COLL}']
+
+    def construct_global_ctx(self):
+        """Constructs the global context"""
+        super().construct_global_ctx()
+        gtx = self.gtx
+        rc = self.rc
+        if "groups" in self.needed_dbs:
+            rc.pi_id = get_pi_id(rc)
+        rc.coll = f"{TARGET_COLL}"
+        try:
+            if not rc.database:
+                rc.database = rc.databases[0]["name"]
+        except:
+            pass
+        colls = [
+            sorted(
+                all_docs_from_collection(rc.client, collname), key=_id_key
+            )
+            for collname in self.needed_dbs
+        ]
+        for db, coll in zip(self.needed_dbs, colls):
+            gtx[db] = coll
+        gtx["all_docs_from_collection"] = all_docs_from_collection
+        gtx["float"] = float
+        gtx["str"] = str
+        gtx["zip"] = zip
+
+    def sout(self):
+        rc = self.rc
+        if rc.filter:
+            collection = key_value_pair_filter(self.gtx["projecta"], rc.filter)
+        else:
+            collection = self.gtx["projecta"]
+
+        projecta = []
+        for projectum in collection:
+            if (not rc.lead) and (not rc.stati):
+                projecta.append(projectum)
+                continue
+            if rc.lead and projectum.get('lead') != rc.lead:
+                continue
+            if rc.stati and projectum.get('status') not in rc.stati:
+                continue
+            projecta.append(projectum)
+
+        selected_projecta = []
+        for p in projecta:
+            if p.get('status') != "proposed":
+                continue
+            selected_projecta.append(p)
+        if selected_projecta:
+            print(f"*************************[Proposed Projecta]*************************")
+            print_projectum(selected_projecta, rc)
+
+        selected_projecta = []
+        for p in projecta:
+            if p.get('status') != "started":
+                continue
+            selected_projecta.append(p)
+        if selected_projecta:
+            print(f"*************************[Started Projecta]**************************")
+            print_projectum(selected_projecta, rc)
+
+        selected_projecta = []
+        for p in projecta:
+            if p.get('status') != "finished":
+                continue
+            selected_projecta.append(p)
+        if selected_projecta:
+            print(f"*************************[Finished Projecta]*************************")
+            print_projectum(selected_projecta, rc)
+
+        selected_projecta = []
+        for p in projecta:
+            if p.get('status') not in ["back_burner", "paused", "cancelled"]:
+                continue
+            selected_projecta.append(p)
+        if selected_projecta:
+            print(f"*************************[Others]************************************")
+            print_projectum(selected_projecta,rc)
```

Comment:
need a carriage return at the end of this line

Reply:
Done.